spec.md

```diff
@@ -136,6 +136,10 @@ a stand-alone event format and stored in the message body.
 A "binary-mode message" is one where the event data is stored in the message
 body, and event attributes are stored as part of message meta-data.
 
+Protocol Bindings MAY use the message body as the request or response body
+directly, or MAY embed the message body in another structure. Protocol Bindings
+MAY batch several message bodies together into a single request or response.
```

Comment:
The "message body" is the thing that contains the wrapped/batch data (meaning it's the transport envelope body), so talking about the message body itself being in another structure isn't quite accurate. The final structure goes into the message body.

What about:
```
Regardless of which mode is used, protocol bindings MAY choose provide
additional formatting and structure to the message body. For example, a
wrapper around the event might be used, or several events could be batched
together into a single message.
```

Reply:
Applied, thanks.